### PR TITLE
feat(antifraud): add new fields

### PR DESCRIPTION
**Pull request by rasecoiac03**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -297,6 +297,98 @@ components:
...
+          type: "string"
+          description: "Nome item"
+          example: "Boné Preto"
+        unitCost:
+          type: "integer"
+          description: "Valor do item em centavos. Ex.: R$ 0,50 => `50` ou R$ 1000,50 => `100050`). **Valor >= 1**"
+          example: 2999
+        quantity:
+          type: "integer"
+          description: "Quantidade de uniades do item"
```
Comment (alissonperez) [inline swagger.yaml]:
```suggestion
          description: "Quantidade de unidades do item"
```

File: swagger.yaml [hunk 2]
```diff
@@ -297,6 +297,98 @@ components:
...
+              type: "string"
+              description: "Id do cliente"
+              example: "123"
+            name:
+              type: "string"
+              description: "Nome do cliente"
+              example: "José Alencar"
+            email:
+              type: "string"
+              description: "Email do cliente"
```
Comment (alissonperez) [inline swagger.yaml]:
```suggestion
              description: "E-mail do cliente"
```

File: swagger.yaml [hunk 3]
```diff
@@ -345,6 +437,36 @@ components:
...
+    TransactionCreationAntifraudResult:
+      type: "object"
+      properties:
+        success:
+          type: "boolean"
+          description: "Transação antifraude feita com sucesso (`true`) ou não (`false`)"
+          example: true
+        status:
+          type: "string"
+          description: "Status da criação da transação. Veja a lista completa [aqui](#tag/Codigos-de-retorno-para-criacao-de-transacoes)"
```
Comment (alissonperez) [inline swagger.yaml]: Is this reference right?

File: swagger.yaml [hunk 4]
```diff
@@ -461,12 +583,47 @@ components:
...
+          description: "Complemento do endereço"
+          example: "Ap 91"
+        neighborhood:
+          type: "string"
+          description: "Bairro do endereço"
+          example: "Vila Clementino"
+        city:
+          type: "string"
+          description: "Cidade do endereço"
+          example: "Rua X"
```
Comment (alissonperez) [inline swagger.yaml]:
```suggestion
          example: "São Paulo"
```

File: swagger.yaml [hunk 5]
```diff
@@ -461,12 +583,47 @@ components:
...
+        state:
+          type: "string"
+          description: "Estado do endereço"
+          example: "Rua X"
```
Comment (alissonperez) [inline swagger.yaml]:
```suggestion
          example: "SP"
```

File: swagger.yaml (hunk 3 above)
Comment (rasecoiac03) [reply swagger.yaml]: Changed. See again please.

**Review by alissonperez — APPROVED** [review]

**Pull request merged**